frontend/src/components/Toggle.jsx

```diff
@@ -22,7 +24,7 @@ export default class Toggle extends Component {
     render() {
         return (
             <a
-                className={cx("Toggle", "no-decoration", { selected: this.props.value }) + " " + (this.props.className||"")}
+                className={cx(styles.toggle, "no-decoration", { [styles.selected]: this.props.value }) + " " + (this.props.className||"")}
```

Comment:
That's kind of the point of CSS modules. There's quite a bit of CSS in `Toggle.css`. CSS modules basically enforce the BEM convention we were already using. This also makes it super easy to split the component out into it's own standalone module.

Inline styles are an alternative, but I'm not a big fan. They break down when you need things like pseudo-classes, and make debugging a pain.

cc @kdoh